firmware/config/boards/microrusefi/compile_mre_f4.sh

```diff
@@ -6,4 +6,6 @@ export EXTRA_PARAMS="-DSHORT_BOARD_NAME=mre_f4"
 
 # export DEBUG_LEVEL_OPT="-O0 -ggdb -g"
 
+export USE_OPENBLT=yes
```

Comment:
let's add same into Proteus F4 and Proteus F7 just to leverage HW CI

not sure which version we actually use for HW CI @mck1117 ?

Reply:
Done for F4 and F7.